README.md

```diff
@@ -66,4 +66,4 @@ We may consider public comment on proposed changes.
 
 ## License
 
-Dedicated to the public domain under CC0-1.0 by Tailscale and contributors.
\ No newline at end of file
+Dedicated to the public domain under CC0-1.0 by Tailscale and contributors.
```

Comment:
I think there was no newline at the end of file so GitHub flags it as a diff. This is fine.